frontend/packages/kubevirt-plugin/integration-tests/tests/vm.wizard.scenario.ts

```diff
@@ -221,4 +227,25 @@ describe('Kubevirt create VM using wizard', () => {
     },
     CLONE_VM_TIMEOUT_SECS,
   );
+
+  it('ICNV-5045 - dont let the user continue If PXE provision source is selected on a cluster without a NAD available', async () => {
+    deleteResources([multusNAD]);
+    const vm = new VMBuilder(getBasicVMBuilder()).build();
+    const wizard = new Wizard();
+    await vm.navigateToListView();
+    await isLoaded();
+    await wizard.openWizard(null);
+    await wizard.fillName(testName);
```

Comment:
can you please use processGeneralStep here

Reply:
I cant use processGeneralStep because it tried to move to the next step but failed because of the error (which I want to verify it existed)

Reply:
but you are clicking on the next button as well to fire up the error. What is the difference?